src/BugsnagUnity/Payload/Breadcrumb.cs

```diff
@@ -9,39 +9,37 @@ namespace BugsnagUnity.Payload
     /// </summary>
     public class Breadcrumb : PayloadContainer, IBreadcrumb
     {
-        private const string UNDEFINED_NAME = "Breadcrumb";
-        private const string NAME_KEY = "name";
-        private const string MESSAGE_KEY = "message";
+        private const string MESSAGE_KEY = "name";
```

Comment:
This seems surprising on the face of it.  Why `name`?

Reply:
added a comment to explain